### ISSUE TITLE
Fail to run site locally

### ISSUE THREAD

**Issue by jotak** [kiali/kiali.io#168]
I haven't investigated deeply but our commands documented in readme don't seem to work anymore.

- `make install` is (almost) OK, I just need to `sudo`
- `make serve` fails with:
  - many errors (maybe not blocking) in `documentation/validations/index.adoc` `WARNING: include file is outside of jail; recovering automatically`
  - Also:

```
Error: Error building site: failed to render pages: render of "taxonomyTerm" failed: "/work/kiali.io/themes/kiali/layouts/_default/list.html.html:14:25": execute of template failed: template: _default/list.html.html:14:25: executing "content" at <where (where $pagesOfSection "Parent" "!=" nil) "Parent.File.Path" .File.Path>: error calling where: runtime error: invalid memory address or nil pointer dereference
make: *** [Makefile:14: serve] Error 255
```

- `make hugo-serve` also fails for me:

```
unknown flag: --mount
See 'docker run --help'.
make: *** [Makefile:33: hugo-serve] Error 125
```

I'm on Fedora 30, running docker 1.13.1, hugo v0.55.6

**Comment by abonas** [comment]
@jotak is this still an issue or can be closed?

**Comment by jotak** [comment]
Looks like it's fixed (fedora 31, hugo 0.59.1)
Warnings are still there, but the site is served & rendered.
